app/controllers/surveys_controller.rb

```diff
@@ -1,4 +1,58 @@
 class SurveysController < ApplicationController
   def send_survey
   end
+
+  def create
+    @survey = Survey.create!(survey_params)
+    questions = []
+
+    params[:questions].each do |question_params|
+      question = Question.create!(
+        text: question_params[:text],
+        question_type: question_params[:question_type].downcase,
+        number: question_params[:id],
+      )
+      questions.push(question)
```

Comment:
Do we have any error handling for if the question model or any other model isn't valid?